### PR TITLE
eZPlatform 3.0 support

### PR DESCRIPTION
**Pull request by ciastektk**

### CODE REVIEW

**Comment by gggeek** [comment]
Interesting! However it is my understanding that many breackages will
happen with ezplatform 3, and I am not sure that it will be possible to
keep the same codebase being compatible all the way up from ez5... Did you
try by any chance to run the full test suite on ezpl3, or are the fixes the
result of only trying to execute a couple of migrations?
If the latter, a good addition would be to add ezpl3 to the test matrix in
the travis config. (And if you feel adventurous, you can check out the
latest master branch and try to run the tests locally using docker...)

File: composer.json [hunk 1]
```diff
@@ -22,39 +24,28 @@
         }
     ],
     "require": {
-        "php": ">=5.6",
+        "php": "^7.3",
```
Comment (gggeek) [inline composer.json]: I am not sure that we can go with php 7.3 as minimum requirement...

File: composer.json [hunk 2]
```diff
@@ -22,39 +24,28 @@
...
         "ext-pdo": "*",
-        "ezsystems/ezpublish-kernel": ">=5.4|>=2014.11",
+        "ezsystems/ezpublish-kernel": "^8.0@dev",
```
Comment (gggeek) [inline composer.json]: same here: we have to append the requirements for 8.0 to the existing 5.4|2014.11 requirements (unless this is meant to be a non-compatible branch...)

**Comment by gggeek** [comment]
Thanks for working out how to update the travis config. I should have some spare time in the upcoming week and be able to try out if it is possible to have a single branch which is compatible with all ez kernel versions or not...

File: composer.json (hunk 1 above)
Comment (damianz5) [reply composer.json]: there is the same dependency on ezp-ee and kernel
https://github.com/ezsystems/ezpublish-kernel/blob/master/composer.json#L10
https://github.com/ezsystems/ezplatform-ee/blob/master/composer.json#L17

**Comment by gggeek** [comment]
ps: seems like bringing back tests is tougher then expected. I created a ticket to track progress: https://github.com/kaliop-uk/ezmigrationbundle/issues/218

**Pull request merged**